addons/portal/static/src/js/portal_name_and_signature.js

```diff
@@ -180,7 +181,7 @@ var NameAndSignature = Widget.extend({
      */
     isSignatureEmpty: function () {
         var signature = this.$signatureField.jSignature('getData');
-        return signature ? this.emptySignature === signature : true;
+        return signature && this.emptySignature ? this.emptySignature === signature : true;
```

Comment:
Feels strange. "If there is not emptySignature saved, then the signature is empty" ?

Reply:
`this.emptySignature` is only set during `start` / `resetSignature`, if `isSignatureEmpty` it should return true because before that point the signature is not in a state to be anything else.